### PR TITLE
use the right repositories for upstream el8 installs

### PR DESCRIPTION
**Pull request by evgeni**
Cherry-pick into:

* [x] Foreman 3.3
* [x] Foreman 3.2
* [x] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

File: guides/common/modules/proc_configuring-repositories.adoc [hunk 1]
```diff
@@ -149,6 +175,10 @@ ifdef::satellite[]
 ----
 endif::[]
 
+ifdef::foreman-el,katello[]
+include::proc_configuring-repositories-el.adoc[]
+endif::[]
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-repositories.adoc:180]: this was removed in https://github.com/theforeman/foreman-documentation/pull/1279/ even after I explicitly said it is required in https://github.com/theforeman/foreman-documentation/pull/1279/files#r885316255

**Comment by evgeni** [comment]
@mjivraja @Lennonka @melcorr followup for #1279 with the right repos for upstream RHEL installs and fix for the fact that #1279 broke the steps needed for installing our own repos on RHEL.

**Comment by melcorr** [comment]
Cherry picks: 
   10f16857..4d18fadb  3.3 -> 3.3
  13c069dd..3db81a63  3.2 -> 3.2
   b2866cae..f7ad2c0d  3.1 -> 3.1

File: guides/common/modules/proc_configuring-repositories.adoc (hunk 1 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-repositories.adoc:180]: I am sorry for this but it was breaking the build. I have replied to the same in #1279 (https://github.com/theforeman/foreman-documentation/pull/1279#discussion_r890961433).

Comment (evgeni) [reply guides/common/modules/proc_configuring-repositories.adoc:180]: It was, yes, and the correct solution was to fix the build instead of removing it.

**Pull request merged**